Gruntfile.js

```diff
@@ -209,6 +209,8 @@ module.exports = function(grunt) {
             'record-screenshots': false
           },
           build: process.env.TRAVIS_BUILD_NUMBER,
+          statusCheckAttempts: 200,
+          pollInterval: 3000,
```

Comment:
I definitely recommend commenting why you chose these options ("something something timeouts").